packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card-banner.tsx

```diff
@@ -120,7 +118,7 @@ const VerificationStatusBanner = ({
                 </Text>
             </div>
         );
-    } else if (is_vanuatu && poi_not_submitted_for_vanuatu_maltainvest) {
+    } else if (is_vanuatu && poa_not_submitted) {
```

Comment:
why do we replace poi_not_submitted_for_vanuatu_maltainvest with poa?

Reply:
Ops. Updated to poi_not_submitted. Thanks for noticing :)